pydis_core/utils/channel.py

```diff
@@ -22,7 +22,10 @@ def is_in_category(channel: discord.TextChannel, category_id: int) -> bool:
     return getattr(channel, "category_id", None) == category_id
 
 
-async def get_or_fetch_channel(bot: Bot, channel_id: int) -> discord.abc.GuildChannel:
+async def get_or_fetch_channel(
+        bot: Bot,
+        channel_id: int
+) -> discord.abc.GuildChannel | discord.abc.PrivateChannel | discord.Thread:
```

Comment:
```suggestion
async def get_or_fetch_channel(
    bot: Bot,
    channel_id: int,
) -> discord.abc.GuildChannel | discord.abc.PrivateChannel | discord.Thread:
```

Reply:
Ugh, pycharm is driving me crazy with this.